test/guard/fixtures_test.go

```diff
@@ -13,15 +13,53 @@ var metadata = &fixtures.MysqlPlainMetaData{
 
 var namedFixtures = fixtures.NamedFixtures[fixtures.MysqlPlainFixtureValues]{
 	{
-		Name:  "fixture_1",
-		Value: fixtures.MysqlPlainFixtureValues{"a97b104f-4d93-4c15-a97a-4e3173e75cde", `{"id": "a97b104f-4d93-4c15-a97a-4e3173e75cde", "effect": "allow", "actions": ["<.+>"], "subjects": ["r:1"], "resources": ["gsl:<.+>"], "conditions": {}, "description": "global - read and write access"}`},
+		Name: "fixture_1",
+		Value: fixtures.MysqlPlainFixtureValues{
+			"a97b104f-4d93-4c15-a97a-4e3173e75cde",
+			`{
+				"description": "global - read and write access",
+				"effect": "allow",
+				"conditions": {},
+				"subjects": ["r:1"],
+				"resources": ["mrn:<.+>"],
+				"actions": ["<.+>"]
+			}`,
+		},
```

Comment:
You dropped the id field, was this intentional?

Reply:
no